android/.gitignore

```diff
@@ -34,6 +34,8 @@ gradle_models.ser
 KMEA/**/assets/ajax-loader.gif
```

Comment:
I don't believe I've ever noticed these paths before. The `**` seems kinda unnecessary and real ugly; can't we move this .gitignore to the assets folder or immediate parent?

(feel free to make this a separate issue for resolution)